routes/createAcc.js

```diff
@@ -0,0 +1,40 @@
+var express = require('express');
+var router = express.Router();
+var userMapper = require('../mappers/userMapper');
+var hashing = require('../config/hashing');
+
+/* GET create account page. */
+router.get('/create/admin', function(req, res) {
+	res.render('createAcc', {user: req.user, err: req.flash('err'), succ: req.flash('succ') });
+});
+
+//Handle POST requests
+//Can find form values at req.body.ELEMNAMEHERE
+router.post('/create/admin', function(req,res){
+	if (validUserParams(req.body)) {
+		var {hash, salt} = hashing.createHash(req.body.inputPassword);
+
+		userMapper.addUser(req.body.inputName, req.body.inputEmail, hash, req.body.type, salt, function(error, result) {
```

Comment:
Need to check passwords equal somewhere  and type is not at element in the ejs so will always be null.  Presumably for now this can just be a constant string: 'admin'